### ISSUE TITLE
Document how to use deploy when git is secured

### ISSUE THREAD

**Issue by ch-stark** [open-cluster-management-io/policy-collection#218]
We might document to add this:
https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.4/html/applications/managing-applications#using-custom-CA-certificates-for-secure-HTTPS-connection

https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.4/html/applications/managing-applications#git-SSH-connection

**Comment by gparvin** [comment]
Yes, we can do this as part of the `--dry-run` feature that was added by @dhaiducek
